app/views/items/_form.html.erb

```diff
@@ -0,0 +1,72 @@
+<%= form_with(model: item) do |form| %>
+  <% if item.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(item.errors.count, "error") %> prohibited this item from being saved:</h2>
+
+      <ul>
+        <% item.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :name, style: "display: block" %>
+    <%= form.text_field :name %>
+  </div>
+
+  <div>
+    <%= form.label :category, style: "display: block" %>
+    <%= form.text_field :category %>
+  </div>
+
+  <div>
+    <%= form.label :location, style: "display: block" %>
+    <%= form.text_field :location %>
+  </div>
+
+  <div>
+    <%= form.label :description, style: "display: block" %>
+    <%= form.text_area :description %>
+  </div>
+
+  <div>
+    <%= form.label :image, style: "display: block" %>
+    <%= form.file_field :image %>
+  </div>
```

Comment:
Would be indeed pretty cool. Unfortunately, we can't access the image via `item.image` since the form has not been submitted yet. I did some research and I think you would need some JavaScript to achieve this (see this [blog post](https://medium.com/@codenode/adding-live-media-previews-to-rails-forms-6c470bc93f3a) for example).
Since this would take some time and other teams are waiting for this story to be completed; I would suggest creating another issue for it

Reply:
For the edit page, this is an improvement though. I added the suggestion :+1:

Reply:
this was added in a similar way in the same place